modules/gapi/test/cpu/gapi_video_tests_cpu.cpp

```diff
@@ -97,4 +97,14 @@ INSTANTIATE_TEST_CASE_MACRO_P(WITH_VIDEO(BuildPyr_CalcOptFlow_PipelineInternalTe
                                       Values(15),
                                       Values(3),
                                       Values(true)));
+
+INSTANTIATE_TEST_CASE_MACRO_P(WITH_VIDEO(BackgroundSubtractorMOG2InternalTestCPU),
+                              BackgroundSubtractorMOG2Test,
+                              Combine(Values(VIDEO_CPU),
+                                      Values("cv/video/768x576.avi"),
+                                      Values("cv/video/1920x1080.avi")));
+
+INSTANTIATE_TEST_CASE_MACRO_P(KalmanFilterInternalTestCPU,
+                              KalmanFilterTest,
+                              Values(VIDEO_CPU));
```

Comment:
I would be great to test different types (32F and 64F) and, perhaps, different initialization structures

Reply:
Done